ci-operator/step-registry/stackrox/install-helm/stackrox-install-helm-commands.sh

```diff
@@ -149,20 +154,53 @@ function install_central_with_helm() {
+    installflags+=('--set' 'central.db.resources.requests.cpu=500m')
+    installflags+=('--set' 'central.db.resources.limits.memory=4Gi')
+    installflags+=('--set' 'central.db.resources.limits.cpu=1')
+    installflags+=('--set' 'scanner.autoscaling.disable=true')
+    installflags+=('--set' 'scanner.replicas=1')
+    installflags+=('--set' 'scanner.resources.requests.memory=500Mi')
+    installflags+=('--set' 'scanner.resources.requests.cpu=500m')
+    installflags+=('--set' 'scanner.resources.limits.memory=2500Mi')
+    installflags+=('--set' 'scanner.resources.limits.cpu=2000m')
+  fi
+
+  if [[ "${ROX_SCANNER_V4:-true}" != "true" ]]; then
+    installflags+=('--set' 'scannerV4.disable=true')
+  else
+    installflags+=('--set' 'scannerV4.disable=false')
+    if [[ "${SMALL_INSTALL}" == "true" ]]; then
+      installflags+=('--set' 'scannerV4.scannerComponent=Enabled')
+      installflags+=('--set' 'scannerV4.indexer.scaling.autoScaling=Disabled')
+      installflags+=('--set' 'scannerV4.indexer.scaling.replicas=1')
+      installflags+=('--set' 'scannerV4.indexer.resources.requests.cpu=600m')
+      installflags+=('--set' 'scannerV4.indexer.resources.requests.memory=1500Mi')
+      installflags+=('--set' 'scannerV4.indexer.resources.limits.cpu=1000m')
+      installflags+=('--set' 'scannerV4.indexer.resources.limits.memory=2Gi')
+      installflags+=('--set' 'scannerV4.matcher.scaling.autoScaling=Disabled')
+      installflags+=('--set' 'scannerV4.matcher.scaling.replicas=1')
+      installflags+=('--set' 'scannerV4.matcher.resources.requests.cpu=600m')
+      installflags+=('--set' 'scannerV4.matcher.resources.requests.memory=5Gi')
+      installflags+=('--set' 'scannerV4.matcher.resources.limits.cpu=1000m')
+      installflags+=('--set' 'scannerV4.matcher.resources.limits.memory=5500Mi')
+      installflags+=('--set' 'scannerV4.db.resources.requests.cpu=200m')
+      installflags+=('--set' 'scannerV4.db.resources.requests.memory=2Gi')
+      installflags+=('--set' 'scannerV4.db.resources.limits.cpu=1000m')
+      installflags+=('--set' 'scannerV4.db.resources.limits.memory=2500Mi')
+    fi
+    if [[ -n "${SCANNER_V4_MATCHER_READINESS:-}" ]]; then
+      # stackrox helm template _metadata.tpl parses 'customize' into values for target matching:
+      # https://github.com/stackrox/stackrox/blob/ae87894195796f9a88295af39a83451dbbb96c51/image/templates/helm/shared/templates/_metadata.tpl#L160-L181
+      # matched for "scanner-v4-matcher" in matcher deployment template:
+      # https://github.com/stackrox/stackrox/blob/62c5f12ba8c3acc0c3d92a71c79221edf25a765f/image/templates/helm/shared/templates/02-scanner-v4-07-matcher-deployment.yaml#L80C9-L80C103
+      # `{{ define "srox._envVars" }}` -> `{{- include "srox.envVars" (list . "deployment" "scanner-v4-matcher" "matcher") | nindent 8 }}`
+      installflags+=('--set' "customize.scanner-v4-matcher.envVars.SCANNER_V4_MATCHER_READINESS=${SCANNER_V4_MATCHER_READINESS}")
```

Comment:
The concept of `SCANNER_V4_MATCHER_READINESS` is not 100% clear to me. I have looked at the stackrox repo and didn't find usage of `SCANNER_V4_MATCHER_READINESS`. Can you please explain how this exactly works?